Plugins/Json/src/DetrayJsonHelper.cpp

```diff
@@ -6,6 +6,13 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+// For whatever reason, this compilation unit does not compile
+// with those assertions and GCC 13. For now just disable the
+// flags here.
+#ifdef _GLIBCXX_ASSERTIONS
+#undef _GLIBCXX_ASSERTIONS
+#endif
```

Comment:
Should this be restricted to gcc13 only then? Does this show up in gcc14 as well?

Reply:
Hmm locally tested with GCC 11 and GCC 14, no issues observed there. So we could restrict this to GCC 13